frontend/public/static/index.css

```diff
@@ -0,0 +1,11 @@
+html, body {
```

Comment:
Easy to add the html & body attributes to the existing styled component, but the root div is not already a styled component. The simplest way to do this is just to add some inline styling since there are only two attributes. What do you guys think?